### PR TITLE
Remove Logrus for log/slog

### LINKED ISSUE #44: Slog through the logging with SLOG
Migrate to log/slog before we get to the point of no return.

### PR DESCRIPTION
**Pull request by jordan-rash**
Closes #44 
This begins the replacement of logrus

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ tasks:
   test:
     deps: [clean]
     cmds:
-      - sudo $GOPATH/bin/ginkgo run -r --randomize-all --vv -race --trace --keep-going ./spec #--cover --coverprofile=.coverage-report.out
+      - sudo ginkgo run -r --randomize-all --vv -race --trace --keep-going ./spec #--cover --coverprofile=.coverage-report.out
```
Comment (jordan-rash) [inline Taskfile.yml:46]: I didnt mean to commit this....its local for me only

File: ui/cmd/nexui/main.go [hunk 2]
```diff
@@ -1,10 +1,70 @@
...
-import nexui "github.com/synadia-io/nex/ui"
+import (
+	"log/slog"
+	"os"
+	"strconv"
+
+	nexui "github.com/synadia-io/nex/ui"
+)
+
+const (
```
Comment (kthomas) [inline ui/cmd/nexui/main.go:11]: Sorry for the nit- I think we may want to audit all of the `const` throughout the codebase to use camelcase.

File: nex/node_linux_amd64.go [hunk 3]
```diff
@@ -2,35 +2,65 @@ package main
...
 	node_preflight.Flag("force", "installs missing dependencies without prompt").Default("false").BoolVar(&NodeOpts.ForceDepInstall)
 	node_preflight.Flag("config", "configuration file for the node").Default("./config.json").StringVar(&NodeOpts.ConfigFilepath)
-	node_up.Action(RunNodeUp)
-	node_preflight.Action(RunNodePreflight)
 
+	ctx := context.Background()
+	opts := slog.HandlerOptions{}
+
+	switch Opts.LogLevel {
+	case "debug":
```
Comment (kthomas) [inline nex/node_linux_amd64.go:23]: May be useful to support `trace` level here also.

File: ui/cmd/nexui/main.go (hunk 2 above)
Comment (jordan-rash) [reply ui/cmd/nexui/main.go:11]: i can live with that

File: nex/node_linux_amd64.go (hunk 3 above)
Comment (jordan-rash) [reply nex/node_linux_amd64.go:23]: thats not a slog default, BUT you can add custom cases.  Ill do that in a different PR

Comment (jordan-rash) [reply nex/node_linux_amd64.go:23]: I openned an issue so i didnt forget.

File: nex/nex.go [hunk 4]
```diff
@@ -14,8 +16,30 @@ var (
 	COMMIT    = ""
 	BUILDDATE = ""
 
-	ncli  = fisk.New("nex", "")
+	blue = color.New(color.FgBlue).SprintFunc()
+
+	ncli = fisk.New("nex", fmt.Sprintf("%s\nNATS Execution Engine CLI Version %s\n", blue(Banner), VERSION))
+	_    = ncli.Author("Synadia Communications")
```
Comment (autodidaddict) [inline nex/nex.go:22]: I really hate that the linter makes us do this, because `ncli.Author(...)` looks so much cleaner.

Comment (jordan-rash) [reply nex/nex.go:22]: There _IS_ a "dumb linter" commit in this PR

**Review by autodidaddict — APPROVED** [review]
LGTM

**Pull request merged**